app/src/main/java/com/randomappsinc/studentpicker/listpage/ListTabsAdapter.java

```diff
@@ -7,34 +7,39 @@
 
 import com.randomappsinc.studentpicker.choosing.NameChoosingFragment;
 import com.randomappsinc.studentpicker.editing.EditNameListFragment;
+import com.randomappsinc.studentpicker.grouping.GroupsFragment;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class ListTabsAdapter extends FragmentStatePagerAdapter {
 
     private String[] tabTitles;
     private String listName;
+    private List<Fragment> fragmentList = new ArrayList<>();
 
-    public ListTabsAdapter(FragmentManager fragmentManager, String listName, String[] tabTitles) {
+    ListTabsAdapter(FragmentManager fragmentManager, String listName, String[] tabTitles) {
         super(fragmentManager, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT);
         this.tabTitles = tabTitles;
         this.listName = listName;
+        adapterSetup();
+    }
+
+    private void adapterSetup() {
```

Comment:
nit: I feel like having adapter in the name is sort of redundant since the method is within the adapter.

```suggestion
    private void setUpFragmentList() {
```